### ISSUE TITLE
[semantic-issue] Add semantics for sklearn KNeighborsClassifier

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#168]
**Description**
Invokes the constructor to instantiate a KNeighborsClassifier object.
Returns an object of KNeighborsClassifier type.

**Minimal running example**
```
from sklearn.neighbors import KNeighborsClassifier

knn_2 = KNeighborsClassifier()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html

**Note (optional)**
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py
